Packet++/src/LdapLayer.cpp

```diff
@@ -72,6 +72,110 @@ namespace pcpp {
 
 	// endregion
 
+	// region LdapResultCode
+
+	const std::unordered_map<LdapResultCode::Value, std::string, EnumClassHash<LdapResultCode::Value>> LdapResultCodeToString{
```

Comment:
make it static? same for the next map.

Reply:
I'm not sure [it really matters](https://stackoverflow.com/a/3709257), but I changed it anyway in this commit: https://github.com/seladb/PcapPlusPlus/pull/1437/commits/c72a2af0fc7b9c3f0dbb9b03f4cf77dc75d5bc9d